src/koopmans/workflows/_koopmans_dscf.py

```diff
@@ -636,15 +636,27 @@ def perform_alpha_calculations(self) -> None:
                         # Calculate an updated alpha and a measure of the error
                         # E(N) - E_i(N - 1) - lambda^alpha_ii(1)     (filled)
                         # E_i(N + 1) - E(N) - lambda^alpha_ii(0)     (empty)
-                        #
-                        # Note that we can do this even from calculations that have been skipped because
-                        # we read in all the requisite information from the output files and .pkl files
-                        # that do not get overwritten
 
                         calcs = [c for c in alpha_dep_calcs + alpha_indep_calcs if c.fixed_band == band]
 
                         alpha, error = self.calculate_alpha_from_list_of_calcs(
                             calcs, trial_calc, band, filled=band.filled)
+                        
+                        # Mixing
+                        alpha = self.parameters.alpha_mixing * alpha + (1 - self.parameters.alpha_mixing) * band.alpha
```

Comment:
Just for the sake of understanding, what is the logic here?
(we can discuss that directly in person tomorrow)